src/features/mutation_minigame/nodes/dna/DNA.gd

```diff
@@ -0,0 +1,54 @@
+extends "res://features/dynamicmenu/DynamicMenu.gd"
+var base_scene = load("res://features/mutation_minigame/nodes/dna/Nucleotide.tscn")
+
+
+func add_base(base_str="BLANK") -> void:
+	var base = base_scene.instantiate()
+	base.base = base_str
+	base.mutation.connect(mutation_handler)
+
+
+func random_dna(length: int):
+	var rng = RandomNumberGenerator.new()
+	var bases: Array = ["A","G","T","C"]
+	
+	var blank_start = base_scene.instantiate()
+	blank_start.mutation.connect(mutation_handler)
+	self.add_child(blank_start)
+	for i in range(length):
+		var base = base_scene.instantiate()
```

Comment:
We seem to be doing this instantiate + mutation.connect + self.add_child a lot. Should it be its function?

Reply:
It literally is lol, I just forgot to actually use it (see [add_base](https://github.com/quaternary-game/quaternary/blob/f71e214b677139504c469e0f1857c44fc55061e4/src/features/mutation_minigame/nodes/dna/DNA.gd#L5))